lsix

```diff
@@ -54,7 +54,11 @@ if [[ ${BASH_VERSINFO[0]} -eq 3 ]]; then
     fi
 fi
 
-if ! command -v montage >/dev/null; then
```

Comment:
Are there actual cases where a person might have `montage` installed but not `identify`? This seems superfluous.

Reply:
I prefered to be cautious. May be some distros like gentoos or openwrt allow to choose to not add it ?

Reply:
I am also a belt-and-suspenders person. My programs often grow due to all the extra checking I do for unlikely or even impossible situations. One of the hardest challenges with `lsix` is to resist that inclination, to let go of perfection, and opt for small, simple, and 99.9% correct. Does that make sense to you?

Reply:
It don't bother me if you trash it.
It doen't make sense neither because it is already there and we end up in the 100% case ;-)